libindy/src/services/wallet/language.rs

```diff
@@ -300,15 +300,16 @@ mod tests {
 
     use super::*;
     use self::rand::{thread_rng, Rng};
+    use rand::distributions::{Alphanumeric, Standard};
     use std::hash::Hash;
     use std::collections::HashSet;
 
     fn _random_vector(len: usize) -> Vec<u8> {
-        thread_rng().gen_iter().take(len).collect()
+        thread_rng().sample_iter(&Standard).take(len).collect()
```

Comment:
note to compare differences between **gen_iter()** and **sample_iter()**

Reply:
From what I can tell, **sample_iter()** is not random. Is this intentional to constrain the test? In any case this shouldn't introduce any problems given it is used to generate randomness in tests.